### PR TITLE
Add keycloak admin console login instructions

### PR DESCRIPTION
**Pull request by arso**
Signed-off-by: Artur Socha <asocha@redhat.com>

### CODE REVIEW

**Comment by arso** [comment]
This is the expected content being displayed in the engine-setup summary (when keycloak enabled)
```          --== SUMMARY ==--
         
[ INFO  ] Starting dwh service
[ INFO  ] Starting Grafana service
[ INFO  ] Restarting ovirt-vmconsole proxy service
          To login to oVirt using Keycloak SSO, enter 'admin@ovirt' as username and the password provided during the setup.
          To login to Keycloak Administration Console enter  'admin' as username and the password provided during the setup 
          Web access for Keycloak Administration Console is enabled at:
              https://dev2.dom/ovirt-engine-auth/admin
          Web access is enabled at:
              http://dev2.dom:80/ovirt-engine
              https://dev2.dom:443/ovirt-engine
          Internal CA fingerprint: SHA256: F6:53:A6:FE:F7:E9:C7:C8:C9:F1:7E:7E:84:33:26:06:7A:70:09:C2:E0:17:02:2F:E5:ED:9A:90:58:B6:53:01
          SSH fingerprint: SHA256:EeSO2+smNuwFr+X5YhTvqxGkfqw4mTri4D6jdMdQOMA
          Engine database resources:
              Database name:      engine_20220713124406
              Database user name: engine_20220713124406
```

**Comment by arso** [comment]
Related BZ for tracking this PR: https://bugzilla.redhat.com/2101474

File: packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/core/ovirt_internal.py [hunk 1]
```diff
@@ -706,21 +706,30 @@ def _results(rc, out):
...
     def _closeup(self):
         self.dialog.note(
             text=_(
-                "Please use the user '{keycloakadmin}' and password specified in "
-                "order to login to Keycloak admin console\n"
-                "Please use the user '{user}' and password specified in "
-                "order to login using Keycloak SSO"
+                "To login to oVirt using Keycloak SSO, "
+                "enter '{user}' as username "
+                "and the password provided during the setup.\n"
```
Comment (didib) [inline packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/core/ovirt_internal.py:718]: Perhaps, to match other texts, "during Setup\n" (no "the", capital 'S', no tailing dot. Not sure I like last one - perhaps we better add dots everywhere - requires more patches).

**Review by didib — APPROVED** [review]

File: packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/core/ovirt_internal.py (hunk 1 above)
Comment (arso) [reply packaging/setup/plugins/ovirt-engine-common/ovirt-engine-keycloak/core/ovirt_internal.py:718]: Fixing "during Setup\n". Thanks!

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**